### PR TITLE
Guided tour of exploration page

### PR DESCRIPTION
**Pull request by danielfdsilva**
I've been exploring user [react tour](https://docs.react.tours/) as a way to create a guided tour of the exploration page. I added a couple of steps just to show case how we could use it. There are a lot more to add, also after the user enter the "analysis mode.

@j08lue @ricardoduplos what do you think of this?

### CODE REVIEW

File: app/scripts/components/exploration/tour-manager.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  gap: ${glsp()};
+  font-weight: ${themeVal('type.base.bold')};
+`;
+
+const tourSteps = [
+  {
+    title: 'Welcome to Exploration',
+    selector: "[data-tour='dataset-list-item']",
+    content:
+      "Each timeline entry represents a dataset, and each of the gray segments represents a data unit in the dataset. This data unit can be a day, month or year, depending on the dataset's time density."
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:49]:
```suggestion
      "Each row represents a dataset, and each of the boxes on the timeline represents a data unit in the dataset. This data unit can be a day, month or year, depending on the dataset's time density."
```

File: app/scripts/components/exploration/tour-manager.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  },
+  {
+    title: 'Playhead',
+    selector: "[data-tour='timeline-head-a']",
+    content:
+      'This is the main timeline playhead which can be used to select the date to view on the map. You can drag it around to change the date.'
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:55]:
```suggestion
      'Move this playhead to select a date to view on the map. You can drag it around or click on the timeline to place it.'
```

File: app/scripts/components/exploration/tour-manager.tsx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    title: 'Timeline',
+    selector: "[data-tour='timeline-interaction-rect']",
+    content: () => (
+      <>
+        To navigate the timeline you can zoom in by pressing the alt key (or
+        option) + the scroll wheel, and click and drag to pan the timeline.
+        <br />
+        Clicking on the timeline will also move the playhead to that date.
+        <br />
+        Go ahead and try it out!
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:72]:
```suggestion
        You can zoom in on the timeline by scrolling while pressing the alt key (or
        option) and click and drag to pan.
        <br />
        Go ahead and try it out!
```

File: app/scripts/components/exploration/tour-manager.tsx [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+      </>
+    )
+  },
+  {
+    title: 'AOI tools',
+    selector: '.mapboxgl-ctrl-top-left',
+    content: () => (
+      <>
+        These tools allow you to draw or upload an AOI to run an analysis on the
+        selected datasets.
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:82]:
```suggestion
        You can calculate a time series (zonal statistics) for your area of interest (AOI). 
        Start that process here by drawing or uploading an AOI.
```

**Review by j08lue — CHANGES_REQUESTED** [review]
As I understood @ricardoduplos, it is a deliberate design choice to keep the controls very simple, without a lot of annotations and explaining labels. 👌 

I think in this case it is necessary to introduce essential features like the ones you highlighted with the guided tour here, @danielfdsilva.

The tour looks great, @danielfdsilva, and I agree with your choice of features to highlight. I am excited to see how our users respond to these.

I made some copy-editing suggestions on the texts - please review them and accept if you agree with my changes.

**Comment by danielfdsilva** [comment]
@j08lue Added another tour when the user draws an AOI. Please give the messages some love :)

File: app/scripts/components/exploration/tour-manager.tsx [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+];
+
+const analysisTourSteps = [
+  {
+    title: 'Analysis',
+    selector: "[data-tour='analysis-message']",
+    content: () => (
+      <>
+        Now that you have an AOI, you can calculate a time series (zonal
+        statistics) for it.
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:96]:
```suggestion
        You can now calculate a time series of zonal
        statistics for your area of interest.
```

File: app/scripts/components/exploration/tour-manager.tsx [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: ${glsp()};
+  font-weight: ${themeVal('type.base.bold')};
+`;
+
+const introTourSteps = [
+  {
+    title: 'Welcome to Exploration',
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:47]:
```suggestion
    title: 'Map layer selection',
```

**Review by j08lue — CHANGES_REQUESTED** [review]
More copy editing for tour messages - I made some of them shorter and used the same type of phrase, presenting the opportunities ("you can ..."). 🤷

File: app/scripts/components/exploration/tour-manager.tsx [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+        Go ahead and try it out!
+      </>
+    )
+  },
+  {
+    title: 'AOI tools',
+    selector: '.mapboxgl-ctrl-top-left',
+    content: () => (
+      <>
+        You can calculate a time series (zonal statistics) for your area of
```
Comment (j08lue) [inline app/scripts/components/exploration/tour-manager.tsx:81]:
```suggestion
        You can calculate a time series of zonal statistics for your area of
```

**Pull request merged**